libsrc/ssdp/SSDPDiscover.cpp

```diff
@@ -275,7 +275,7 @@ int SSDPDiscover::discoverServices(const QString& searchTarget, const QString& k
 					}
 					else
 					{
-						_services.insertMulti(mapKey, service);
```

Comment:
@m-seker The original Code was the _services was a QMap.
In case of "skipDuplicates" the identified service records were added to the single key, otherwise multiple keys were added using insertMulti.

With the latest change to QMultiMap and this change, how will duplicate key entries be avoided?
This change will even make the "if ( _skipDupKeys )" check obsolete....

Reply:
You are right. I blindly changed it :) Already fixed, check again please.